README.md

```diff
@@ -1,5 +1,116 @@
+[![GitHub tag](https://img.shields.io/github/tag/reearth/reearth.js.svg)](https://GitHub.com/reearth/reearth.js/tags/)
+[![Github all releases](https://img.shields.io/github/downloads/reearth/reearth.js/total.svg)](https://GitHub.com/reearth/reearth.js/releases/)
+[![GitHub contributors](https://img.shields.io/github/contributors/reearth/reearth.svg)](https://GitHub.com/reearth/reearth/graphs/contributors/)
+[![](https://img.shields.io/github/contributors/reearth/reearth-web?label=front-end%20contributors)](https://GitHub.com/reearth/reearth-web/graphs/contributors/)
+[![](https://img.shields.io/github/contributors/reearth/reearth-backend?label=back-end%20contributors)](https://GitHub.com/reearth/reearth-backend/graphs/contributors/)
+[![Generic badge](https://img.shields.io/badge/discussion-welcome-green.svg)](https://github.com/reearth/reearth/discussions)
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+<!-- TODO: change this link later -->
+  <a href="https://github.com/reearth/reearth">
+    <img src="./public/reearth-logo.svg" alt="Logo" width="300" height="300">
+  </a>
+
+  <p align="center">
+    Re:Earth is a state-of-the-art WebGIS with expandability through plug-ins and the ease of running in a browser.
+    <br />
+    <a  href="https://docs.reearth.io/"><strong>Documents >></strong></a>
+    &emsp;&emsp;&emsp;&emsp;
+    <a href="https://docs.reearth.io/oss"><strong>OSS Webpage >></strong></a>
+    <br />
+    <!-- TODO: here -->
+    <a href="">View Demo</a>
+    ·
+    <a href="https://github.com/reearth/reearth/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/reearth/reearth/discussions">Request Feature</a>
+  </p>
+</p>
+
+Re:Earth consist with 2 repositories. Don't forget to check our [front-end](https://github.com/reearth/reearth-web) and [back-end](https://github.com/reearth/reearth-backend) repositories👍
+
+[![reearth-ui](./public/reearth-ui.png)](https://reearth.io)
+
+<!-- ABOUT THE PROJECT -->
+## Features
+🔌 Higy extensible with plugins system
+
+💻 Super handy with running on the browser
+
+💪 Support standard GIS data formats(CSV, KML, CZML, GeoJSON, shapefile)
+
+📢 Easily make the project public
+
+✨ Freely style the map
+
+
+## Built With
+* [React](https://github.com/facebook/react)
+* [TypeScript](https://github.com/microsoft/TypeScript)
+* [Go](https://github.com/golang/go)
+* [GraphQL](https://github.com/graphql)
+* [MongoDB](https://www.mongodb.com/)
+* [WebAssembly](https://webassembly.org/) + [QuickJS](https://github.com/bellard/quickjs)
+
+
+### Prerequisites
+
+**Front-end**
+* Node.js: Latest version of LTS
+
+**Back-end**
+* Golang +1.16
+* Docker
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+
+_For more examples, please refer to the [Documentation](https://docs.reearth.io)_
+
+
+
+<!-- ROADMAP -->
+## Roadmap
+
+See the [Roadmap](https://github.com/reearth/reearth/projects/1) for a list of proposed features (and known issues).
```

Comment:
If the roadmap has proposed features, the features have only been suggested and not committed to being developed. If that is what you mean this is fine.
But if the features listed are on track to being developed (for sure, or 90% sure :P) this might be better:
```suggestion
See the [Roadmap](https://github.com/reearth/reearth/projects/1) for a list of planned features and known issues.
```

Reply:
The planned proposal will be put in the `Q? 202?` column. The good proposals which are not sure when developed are in `Influential proposals`. In this case, it contains both. Which is better?